fixieai/agents-ts/package.json

```diff
@@ -19,7 +19,7 @@
     "prepublishOnly": "npm run build",
     "build": "tsc -p . && chmod a+x dist/serve-bin.js",
     "start:dev": "npx nodemon",
-    "bin:dev": "npx ts-node src/serve-bin.ts",
+    "bin:dev": "npx ts-node src/serve-bin.ts --userStorageApiUrl https://app.fixie.ai/api/userstorage",
```

Comment:
@petersalas `src/backend/fixie/frame/views_deployments.py` will have to be updated to pass this flag.

Reply:
And then hardcode the `/api/userstorage` pathname? I'm fine with that, assuming you think it'll be relatively stable.